cypress/integration/index/cookie_banner.spec.js

```diff
@@ -0,0 +1,121 @@
+describe('Cookie banner', () => {
+  before(() => {
+    cy.visit('/')
+    cy.checkBanner()
```

Comment:
Do we want to check this at every test or is it just in the cookies test because there's another banner here at play? 

Also, I feel it'd be helpful to disambiguate this command with `checkEnvBanner` or similar? I thought it was checking the cookie banner in this instance

Reply:
Makes sense thanks. I'll change the name to checkEnvBanner(), which makes more sense.

As for disabling JS, it was difficult to do but not impossible. The workaround / hack e.g. https://github.com/cypress-io/cypress/issues/1611#issuecomment-749406961 disabled a chrome browser configuration or injection of js. This lead to inconsistencies with how the app actually works, so I just tested with JS enabled. As cypress needed JS to work.